blitzstudios/sleeper-mini
# Removing local version of sleeper components and replacing them with @sleeperhq package.

### Description

To minimize required updates to this project when core sleeper components change, we've now abstracted out all the dev server / sleeper UI to a separate package, hosted on NPM. This package is located here: https://www.npmjs.com/package/@sleeperhq/mini-core

### How To Test

1. Start this template project as usual, making sure to run `yarn` and `pod install`.
2. Connect to a running sleeper app and verify functionality remains unchanged.
